ImprovedAnvil/lib/spells/03_patches_vanilla.tph

```diff
@@ -170,6 +430,152 @@ COPY_EXISTING ~SPPR350.spl~ ~override/SPPR350.spl~
   LPF DELETE_EFFECT INT_VAR match_opcode = 321 STR_VAR match_resource=~SPPR416~ END
   LPF DELETE_EFFECT INT_VAR match_opcode = 321 STR_VAR match_resource=~BDAURAFE~ END
 
+// Remove summon animals from Clerics
```

Comment:
I presume you've removed the flags
but the spells are still potentially present in the joinable npcs' spellbooks. these need to be removed as well. there are plenty of examples in patches_joinable.tph

Reply:
okay, I will take a look at that

Reply:
Spells removed from Anomen, Aerie and Viconia, they had them in spellbook, though no instances ready to use.